ci-operator/config/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main.yaml

```diff
@@ -39,8 +39,10 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: unit
-  commands: make gotest
+- as: functional
+  commands: |
+    export GOFLAGS=
```

Comment:
You are getting some permission denied in this test.
One thing that you can do is to copy the code and run from another dir:
```
mkdir -p ../operator && cp -r . ../operator
cd ../operator
make test GINKGO_ARGS='--no-color'
```

Reply:
yeah, not getting if this is expected though[1].
I see `nova-operator` does something similar [2], so I think I'm doing something wrong here :/

[1] https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/39368/rehearse-39368-pull-ci-openstack-k8s-operators-manila-operator-main-functional/1658493606723325952/artifacts/test/build-log.txt
[2] https://github.com/openshift/release/blob/master/ci-operator/config/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-main.yaml#L49

Reply:
It something different in Makefile. If you search for `rm -f api/bases` in their logs, you will not find. Which is the operation that is failing in your test.
You can try to fix on the Makefile, or make a copy of the src code content

Reply:
ack thanks @viroel, I didn't consider that part. I did the change you suggested, let's see now the CI in action!